aspnet/conceptual-overview/aspnet.rst

```diff
@@ -57,19 +77,21 @@ ASP.NET Core applications are defined using a public ``Startup`` class:
       }
   }
 
-The ``ConfigureServices`` method defines the services used by your application and the ``Configure`` method is used to define what middleware makes up your request pipeline. See :doc:`/fundamentals/startup` for more details.
+- :doc:`ConfigureServices </fundamentals/startup>`  defines the services used by your app (such as the ASP.NET MVC Core framework, Entity Framework Core, Identity, etc. )
+- :doc:`Configure </fundamentals/startup>` defines the :doc:`middleware </fundamentals/middleware>` in the request pipeline
+- See :doc:`/fundamentals/startup`
 
 Services
 --------
 
-A service is a component that is intended for common consumption in an application. Services are made available through dependency injection. ASP.NET Core includes a simple built-in inversion of control (IoC) container that supports constructor injection by default, but can be easily replaced with your IoC container of choice. See :doc:`/fundamentals/dependency-injection` for more details.
+A service is a component that is intended for common consumption in an application. Services are made available through dependency injection. ASP.NET Core includes a simple built-in inversion of control (IoC) container that supports constructor injection by default, but can be easily replaced with your IoC container of choice. In addition to the loose coupling benefit DI provides, it also makes services available throughout your app. For example, :doc:`Logging </fundamentals/logging>` is available throughout your app. See :doc:`/fundamentals/dependency-injection` for more details.
```

Comment:
**Edit fourth sentence.** In addition to its loose coupling benefit, DI also makes services...